src/modeler/AutoRest.Swagger/Validation/SummaryDescriptionValidation.cs

```diff
@@ -37,7 +37,7 @@ public class SummaryDescriptionValidation: TypedRule<Operation>
 
         public override IEnumerable<ValidationMessage> GetValidationMessages(Operation operation, RuleContext context)
         {
-            if(operation.Description != null && operation.Summary != null && operation.Description.Trim().Equals(operation.Summary.Trim(), StringComparison.CurrentCultureIgnoreCase))
+            if(operation.Description != null && operation.Summary != null && operation.Description.Trim().Equals(operation.Summary.Trim(), StringComparison.OrdinalIgnoreCase))
```

Comment:
I think you meant `null-conditional  --> ?.` . Agreed that it makes it comparing against a bool harder, as the result of the expression would be a `Nullable<bool>`.